### ISSUE TITLE
X offset doesn't work if insert() position is "bottom"

### ISSUE THREAD

**Issue by therealsamsquanch** [Intervention/image#1030]
For some reason, I can't change the image's X position inside the canvas if I set insert() to "bottom". I tried larger integers and negative integers but no luck, it's ignored. To be clear, the image is correctly positioned at the bottom of the canvas, but the X offset doesn't work. I can set Y's position just fine.

If I set insert() to "bottom-right", both X and Y work.

Tried with GD and Magick.

I'm using the latest release of Intervention Image, 2.5.1.

```php
use Intervention\Image\ImageManagerStatic as Image;

Image::configure(array('driver' => 'imagick'));

$canvas = Image::canvas(400, 600);

$logo = Image::make('/my_logo.png');
$logo->resize(150, 150, function ($constraint) {
            $constraint->aspectRatio();
            $constraint->upsize();
});
$logo->rotate(-90);

$canvas->insert($logo, 'bottom', 170);
```

**Comment by electronick86** [comment]
I have the same problem, I'm surprised that the offset cannot be used with "centered" positions.

The problem seems documented (and solved) here #792 but still present with 2.5.1

**Comment by olivervogel** [comment]
This issue should no longer occur in version 3.
